src/commands/migrate-latest.ts

```diff
@@ -21,6 +22,15 @@ class MigrateLatest extends Command {
     })
   };
 
+  /**
+   * Started event handler.
+   */
+  onStarted = async (result: OperationResult) => {
+    await printLine(bold(` ▸ ${result.connectionId}`));
+    await printLine(bold(' ▸ DRY RUN STARTED'));
```

Comment:
Think these two lines should look a bit different as they're trying to represent different things.

Would be better if we can use something other than ` ▸` for dry run log.
This will help easily distinguish `connection id` from the `process`.

```suggestion
    await printLine(bold(` ▸ ${result.connectionId}`));
    await printLine(' • DRY RUN STARTED');
```